code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2222,7 +2222,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		if(prob(10))
 			stored_teleports += rand(2,6)
 			if(prob(70))
-				M.vomit()
+				M.vomit(vomit_type = VOMIT_PURPLE)
```

Comment:
Ditto

Reply:
this is more of a refactor that removes a snowflake check for this specific reagent and instead allows a vomit_type parameter anywhere